cmds/recv.c

```diff
@@ -42,7 +42,10 @@ static int server_socket(struct opts *opts)
 	if (sock < 0)
 		pr_err("socket creation failed");
 
-	setsockopt(sock, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on));
+	if (setsockopt(sock, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on)) != 0)
+		pr_err("socket setting failed");
```

Comment:
`pr_err` means it would exit if fails.  Have you run the tests with this change?  Maybe recv failures are relevant..

Reply:
hmm.. I thought it should exit if there was any problem with `setsockopt`.
If you don't recommend this way, I will amend commit not using `pr_err`.

Also, all unitests had passed.

`./runtest.py` result is like below, pretty same as master branch's result:

**PR result**

```
$ ./runtest.py

...
141 recv_basic          : OK OK OK OK OK OK OK OK OK OK
142 recv_multi          : OK OK OK OK OK OK OK OK OK OK
143 recv_kernel         : SK SK SK SK SK SK SK SK SK SK
...
150 recv_event          : SK SK SK SK SK SK SK SK SK SK
151 recv_runcmd         : NG NG NG NG OK NG NG NG NG NG
...
167 recv_sched          : SK SK SK SK SK SK SK SK SK SK
```

**master branch (original) result**

```
$ ./runtest.py
...
141 recv_basic          : OK OK OK OK OK OK OK OK OK OK
142 recv_multi          : OK OK OK OK OK OK OK OK OK OK
143 recv_kernel         : SK SK SK SK SK SK SK SK SK SK
...
150 recv_event          : SK SK SK SK SK SK SK SK SK SK
151 recv_runcmd         : NG NG NG NG NG NG NG NG NG NG
...
167 recv_sched          : SK SK SK SK SK SK SK SK SK SK
```

Reply:
Sorry for the late reply.  I think `setsockopt` here is a hint so it's ok to fail, please change it to `pr_warn` instead.

Reply:
Thank you for your comment :) I just changed it to `pr_warn` and ran tests.